_includes/meta.html

```diff
@@ -61,6 +61,115 @@
     gtag('js', new Date());
 
     gtag('config', '{{ site.ga.gtag }}');
+
+    function checkDomainAvailability(requestedDomainId="domain-input", isDomainChoosing=false, alertId="usa-search--domain_message") {
+
+      // Cancel the default action
+      event.preventDefault();
+
+      const baseUrl = "https://manage.get.gov/api/v1/available/?domain="
+
+      let requested_domain = document.getElementById(requestedDomainId).value
+
+      if (requested_domain === null || requested_domain.length === 0) {
+          render_result(``)
+      }
+
+      let searchEndpoint = new URL("", baseUrl + requested_domain);
+      fetch(searchEndpoint).then(function(reply) {
+          return reply.json()
+      }).then(function(response) {
+          let requested_domain_display_string = requested_domain
+              if (!requested_domain_display_string.endsWith('.gov')) {
+                  requested_domain_display_string += '.gov'
+              }
+          
+          let successMessage = `
+          <div class="usa-alert usa-alert--success usa-alert--slim" role="alert">
+              <div class="usa-alert__body">
+                  <p class="usa-alert__text">
+                      That domain is available!
+                      <a class="usa-link" href="{{ '/domains/before/' | url }}">
+                          Read about next steps to take before requesting your .gov domain.
+                      </a>
+                  </p>
+              </div>
+          </div>
+          `
+
+          let errorMessage = null
+          if (isDomainChoosing) {
+              errorMessage = document.getElementById("grid-row--domain_message-choosing");
+              if (errorMessage) {
+                  // Check if the 'display-none' class is not already present
+                  if (!errorMessage.classList.contains("display-none")) {
+                      errorMessage.classList.add("display-none");
+                  }
+              }
+
+              successMessage = `
+              <div class="usa-alert usa-alert--success usa-alert--slim" role="alert">
```

Comment:
@zandercymatics I can't believe I'm just now noticing this, but the success and error alerts on this page and on the home page check availability don't use our .gov custom color alerts. Can we make these alerts match the ones on the registrar?